src/Stripe.net/Stripe.net.csproj

```diff
@@ -19,6 +19,7 @@
     <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.2' ">$(PackageTargetFallback);netcoreapp1.0</PackageTargetFallback>
     <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.2' ">1.6.1</NetStandardImplicitPackageVersion>
     <SignAssembly>True</SignAssembly>
+    <CodeAnalysisRuleSet>Settings.stylecop</CodeAnalysisRuleSet>
```

Comment:
I don't think you need this one.

Reply:
ah yes sorry forgot that I copy/pasted that everywhere to try to make it work :p